pkg/apis/admission/fuzzer/fuzzer.go

```diff
@@ -17,10 +17,20 @@ limitations under the License.
 package fuzzer
 
 import (
+	fuzz "github.com/google/gofuzz"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	runtimeserializer "k8s.io/apimachinery/pkg/runtime/serializer"
 )
 
 // Funcs returns the fuzzer functions for the admission api group.
 var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
-	return []interface{}{}
+	return []interface{}{
+		func(s *runtime.RawExtension, c fuzz.Continue) {
+			cm := &v1.ConfigMap{}
```

Comment:
hrmm, I don't love taking a dependency on the v1 package just for this. can we assign a fuzzed Unstructured object to Object, or a json blob to the Raw subfield instead?

Reply:
Humm. Both `Object` and `Raw` are appropriate for different cases. I'll go with `Unstructured` into `Object` since that is more reflective of what is needed once everything is fully marshaled to typed objects. If anyone needs to exercise raw cases they can set up a fuzzer for that case in the future.

Reply:
Looks like the test package also as a RawExtension fuzzer https://github.com/kubernetes/kubernetes/blob/a520302fb4673e595fcb70d2a4db26598371be92/pkg/api/testing/fuzzer.go#L67. But I'm not sure if should add a dependency on that from agnhost image code.